client/src/pages/Profile.js

```diff
@@ -48,60 +49,70 @@ const useStyles = makeStyles((theme) => ({
     }
 }))
 
-const tempData = [
-    {
-        image: 'tattoo-1.png',
-        noSketches: '24',
-        title: 'Lion tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '150'
-    },
-    {
-        image: 'tattoo-3.png',
-        noSketches: '24',
-        title: 'Cat tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '200'
-    },
-    {
-        image: 'tattoo-7.png',
-        noSketches: '24',
-        title: 'Lion tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '90'
-    },
-]
+const user_id = 1
 
 export default function Profile() {
     const classes = useStyles()
     const [activeTab, setActiveTab] = useState(0)
+    const [contests, setContests] = useState([])    
+    const [inProgressContestCards, getInProgressContestCards] = useState([])
+    const [completedContestCards, getCompletedContestCards] = useState([])
 
     const handleTabChange = (event, newActiveTab) => {
         setActiveTab(newActiveTab)
     }
-
-    const inprogressContests = tempData.map((contest, index) => (
-        <ContestCard 
-            key={index}
-            image={contest.image} 
-            noSketches={contest.noSketches} 
-            title={contest.title}
-            description={contest.description}
-            prizeAmount={contest.prizeAmount}
-        />
-    ))
-
-    const completedContests = tempData.map((contest, index) => (
-        <ContestCard 
-            key={index}
-            image={contest.image} 
-            noSketches={contest.noSketches} 
-            title={contest.title}
-            description={contest.description}
-            prizeAmount={contest.prizeAmount}
-        />
-    )).reverse()
-
+    const getData = () => {
+        setContests(getOwnedContests(user_id, (data) => {
+            setContests(data) // Sets contests equal to return from get request
+        }, (error) => {
+            // onError
+            if (error instanceof RequestError && error.response.status === 400) {
+                console.log(error.response.json())
+            } else {
+                console.log("unexpected error")
+            }
+        }))
+    }
+    useEffect(() => { // Only runs once when first rendering
+        getData() 
```

Comment:
i'm guessing this is returning a React warning in the console logs because `getData` is not a decency here. Since `getData` only ever gets called here, you can actually move the function into this useEffect block

```
useEffect(() => {
  const getData = () => {
      ...
  }

  getData();
}, [])
```